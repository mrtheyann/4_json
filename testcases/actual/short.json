{"metadata":{"name":""},"nbformat":3,"nbformat_minor":0,"worksheets":[{"cells":[{"cell_type":"heading","level":1,"metadata":{},"source":["\u0417\u0430\u0434\u0430\u0447\u04301"]},{"cell_type":"code","collapsed":false,"input":["fromnumpyimport*\n","importmatplotlib.pyplotasplt\n","%matplotlibinline"],"language":"python","metadata":{},"outputs":[],"prompt_number":96}]}]}