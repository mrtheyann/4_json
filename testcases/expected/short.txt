{
    "metadata": {
        "name": ""
    },
    "nbformat": 3,
    "nbformat_minor": 0,
    "worksheets": [
        {
            "cells": [
                {
                    "cell_type": "heading",
                    "level": 1,
                    "metadata": {},
                    "source": [
                        "Задача1"
                    ]
                },
                {
                    "cell_type": "code",
                    "collapsed": false,
                    "input": [
                        "fromnumpyimport*\n",
                        "importmatplotlib.pyplotasplt\n",
                        "%matplotlibinline"
                    ],
                    "language": "python",
                    "metadata": {},
                    "outputs": [],
                    "prompt_number": 96
                }
            ]
        }
    ]
}